backwards-compatability/node10/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "backcompat-node10",
-  "version": "0.19.0",
+  "version": "0.20.0",
   "private": true,
   "description": "Backwards compatability app for node8 types and the OpenTelemetry Node.js SDK",
   "main": "index.js",
   "scripts": {
     "test:backcompat": "tsc --noEmit index.ts"
   },
   "dependencies": {
-    "@opentelemetry/sdk-node": "0.19.0",
-    "@opentelemetry/tracing": "0.19.0"
+    "@opentelemetry/sdk-node": "^0.20.0",
```

Comment:
is it intended to use `^`here and other backward compat tests?

Reply:
I had the same thoughts but figured the approach taken has been. For examples and back compat tests use the auto update (to identify breaks). But for release components it's pinned... Which sort of makes sense.

We will still need to update with each release just in case we make some sort of forward breaking changes. (ie. add and use new functions in a later version that are not available in older ones) -- so we still rev forward in this case rather than just leaving these auto updates using ^0.20.0 etc.

I'm fine either way (with or without the ^) for the ones that are defined in this PR -- as long as we continue to update moving forward.

Reply:
😡 lerna keeps adding `^` and then dependabot removes it. We're in a constant vicious cycle

Reply:
I think you would need to add `--exact` to `lerna version` command to ensure its not using `^`?